### DISCUSSION TITLE
Where are your issues ?

### DISCUSSION THREAD

**Discussion by sammou00** [HYF-Class21/workflows-group3-intro#4] [Q&A]
Where are your issues? How can you make a merge without an issue?
Check group 1
[group 1 ](https://github.com/HYF-Class21/workflows-group1-intro/issues)

**Comment by sammou00** [comment]
@alisanlii  I have noticed, you added info about Greek, you suppose to add info about yourself.

**Reply by alisanlii** [reply]
Hi Samir, I thought we can choose the subject spontaneously. I will change it tomorrow after make a meeting with t=my group.

**Comment by sammou00** [comment]
@alisanlii @RabitaMaharjan @XinGITLou @Juharzeynu 

1- You should have an issue for each pull request , meaning you need 4 issues for each member info + 1 for update `README.md`
2- we don't add `pull requests` to the `project board` we only add` issues`.
3- you should link each `PR` with its issue.
4- the issue will be closed when the `PR` is merged
